frontend/packages/console-shared/src/components/formik-fields/SyncedEditorField.tsx

```diff
@@ -120,7 +155,15 @@ const SyncedEditorField: React.FC<SyncedEditorFieldProps> = ({
     setDisabledFormAlert(formContext.isDisabled);
   }, [formContext.isDisabled]);
 
-  return (
+  React.useEffect(() => {
+    if (loaded) {
+      setLastViewedEditorType(activeEditorType);
+      setFieldValue(name, activeEditorType);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Add a comment why the linting is disabled here.

Reply:
done